Create ui-c8y-1019-9-1-fix-add-pipe-to-generate-asset-group-device-map-marker-pop.md (#1808)
SoftwareAG/c8y-docs: content/change-logs/ui-c8y-1019-9-1-fix-add-pipe-to-generate-asset-group-device-map-marker-pop.md

---
date: ""

> title: Fixed map marker popups link to assets and devices

Reviewer: ```suggestion
title: Map marker popup correctly links to assets and devices
```

> title: fix: Add pipe to generate asset/group/device map marker pop… (#5771) [GRAFT][release/cd] (#5834)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Add map marker popups for assets, groups, and devices
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57991
version: 1019.9.1
---

> The marker popup now referencing correctly to an asset or a group if the link in the popup is clicked.

Reviewer: ```suggestion
The link in the marker popup of the map now correctly references the respective asset or a group.
```

> fix: Add pipe to generate asset/group/device map marker pop… (#5771) [GRAFT][release/cd] (#5834)

Author: The Claude AI suggested to use the following body:
```suggestion
Added a pipe to generate map marker popups for assets, groups, and devices on maps. The popups display identifying information about the asset, group, or device.
```
